### PR TITLE
re-rendered with conda-build 3.17.7, conda-smithy 3.2.9, and conda-forge-pinning 2019.01.21; update to 1.9.3

### PR DESCRIPTION
**Pull request by notestaff**
Checklist
* [X] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by notestaff** [comment]
@conda-forge/core @conda-forge/stack Do these warnings need fixing?

WARNING: Disagreement in get_linkages(filename=/home/conda/feedstock_root/build_artifacts/stack_1548217054376/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/bin/stack, resolve_filenames=True, recurse=False, sysroot=, envroot=/home/conda/feedstock_root/build_artifacts/stack_1548217054376/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place, arch=native):
 lief: {'$RPATH/libm.so.6', '$RPATH/libgmp.so.10', '$RPATH/libutil.so.1', '$RPATH/librt.so.1', '$RPATH/libpthread.so.0', '$RPATH/libz.so.1', '$RPATH/libgcc_s.so.1', '$RPATH/libdl.so.2', '$RPATH/libc.so.6'}
pyldd: {'$RPATH/libgmp.so.10', '/lib64/libz.so.1', '/lib64/librt.so.1', '/lib64/libutil.so.1', '/lib64/libdl.so.2', '/lib64/libm.so.6', '/lib64/libgcc_s.so.1', '/lib64/libpthread.so.0', '/lib64/libc.so.6'}
  (using lief)
WARNING (stack,bin/stack): did not find - or even know where to look for: $RPATH/libm.so.6
WARNING (stack,bin/stack): did not find - or even know where to look for: $RPATH/libgmp.so.10
WARNING (stack,bin/stack): did not find - or even know where to look for: $RPATH/libutil.so.1
WARNING (stack,bin/stack): did not find - or even know where to look for: $RPATH/librt.so.1
WARNING (stack,bin/stack): did not find - or even know where to look for: $RPATH/libpthread.so.0
WARNING (stack,bin/stack): did not find - or even know where to look for: $RPATH/libz.so.1
WARNING (stack,bin/stack): did not find - or even know where to look for: $RPATH/libgcc_s.so.1
WARNING (stack,bin/stack): did not find - or even know where to look for: $RPATH/libdl.so.2
WARNING (stack,bin/stack): did not find - or even know where to look for: $RPATH/libc.so.6

**Comment by scopatz** [comment]
The warnings should be fixed

**Comment by notestaff** [comment]
@scopatz Now it's down to one warning:
"WARNING (stack,bin/stack): Needed DSO / not found in any CDT/compiler package, nor the whitelist?!"
Is there a way to fix that?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,9 +19,16 @@ source:
...
+  number: 1000
+  ignore_run_exports:
+      - libstdcxx-ng
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - gmp >=5.0.1,<7  # [not win]
```
Comment (isuruf) [inline recipe/meta.yaml]: This should be just `- gmp`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -19,9 +19,16 @@ source:
...
   run:
     - gmp >=5.0.1,<7  # [not win]
```
Comment (isuruf) [inline recipe/meta.yaml]: Same here

File: recipe/meta.yaml [hunk 3]
```diff
@@ -19,9 +19,16 @@ source:
   sha256: {{ winsha256 }}  # [win]
 
 build:
-  number: 1
+  number: 1000
+  ignore_run_exports:
+      - libstdcxx-ng
```
Comment (isuruf) [inline recipe/meta.yaml]: Why ignore run exports here?

**Comment by notestaff** [comment]
@isuruf Is there a way to specify     - {{ compiler('c') }} in the host requirements?  stack needs gcc at runtime .

**Comment by notestaff** [comment]
@conda-forge/core Something changed in the latest re-render, such that stack always fails with the error that -lgmp is not found, even though gmp is a host and run requirement.  Any thoughts on what that could be?

**Comment by LaurentRDC** [comment]
This PR is not relevant anymore. If you need this specific version, please re-open this PR.

**Pull request closed without merging**